image/image.go

```diff
@@ -121,12 +121,37 @@ func localSnaps(tsto *ToolingStore, opts *Options) (*localInfos, error) {
 	}, nil
 }
 
+func validateNoParallelSnapInstances(snaps []string) error {
+	for _, snapName := range snaps {
+		_, instanceKey := snap.SplitInstanceName(snapName)
+		if instanceKey != "" {
+			return fmt.Errorf("cannot use snap %q, parallel snap instances are unsupported",
+				snapName)
+		}
+	}
+	return nil
+}
+
+func validateNonLocalSnaps(snaps []string) error {
```

Comment:
Can you add a comment to this function. I also find it a bit annoying to read since the words together make it harder to distinguish `Non` from `No`.